tests/test_arcaflow_plugin_utilities.py

```diff
@@ -35,6 +39,13 @@ def test_functional(self):
         self.assertEqual("success", output_id)
         self.assertIsNotNone(output_data.timestamp)
 
+        output_id, output_data = utilities_plugin.wait(
+            params=utilities_plugin.WaitInput(wait_time_ms=500), run_id="plugin_ci"
+        )
```

Comment:
Is there (supposed to be) any relationship between the `wait_time_ms` value specified here and the value in line 19?

Reply:
Separate test cases, I think, though encapsulating multiple test cases in one function obscures the logic a bit. Though there's no reason they couldn't be the same, especially since 19 is just a serialization test.